### PR TITLE
Add release note for 2.7.0.

### PR DESCRIPTION
**Pull request by codelipenghui**

### CODE REVIEW

**Review by Jennifer88huang — APPROVED** [review]

**Review by jiazhai — APPROVED** [review]

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,11 +1,180 @@
...
+- Support produce messages with transaction in batch [#8415](https://github.com/apache/pulsar/pull/8415)
+- Register transaction metadata before send or ack messages [#8493](https://github.com/apache/pulsar/pull/8493)
+- Expose transaction interface [#8505](https://github.com/apache/pulsar/pull/8505)
+- Guarantee transaction metadata handlers connected [#8563](https://github.com/apache/pulsar/pull/8563)
+- Add the batch size in transaction ack command [#8659](https://github.com/apache/pulsar/pull/8659)
+- Transaction Log implementation [#8658](https://github.com/apache/pulsar/pull/8658)
+
+#### Topic policy
+
+- Support set message TTL on topic level [#7738](https://github.com/apache/pulsar/pull/7738)
```
Comment (Huanli-Meng) [inline site2/website/release-notes.md:30]: support + doing, so change "support set" to "support setting", same comment for the below.

File: site2/website/release-notes.md [hunk 2]
```diff
@@ -1,11 +1,180 @@
...
+- Allow specifying state storage url for Source/Sink localrun [#7930](https://github.com/apache/pulsar/pull/7930)
+- Enable function worker JVM metrics to be reported via Prometheus [#8097](https://github.com/apache/pulsar/pull/8097)
+- Add ability to specify EnvironmentBasedSecretsProvider in LocalRunner [#8098](https://github.com/apache/pulsar/pull/8098)
+- Added ability to specify secrets class in localrunner builder [#8127](https://github.com/apache/pulsar/pull/8127)
+- Add access to the current message from the function context [#8290](https://github.com/apache/pulsar/pull/8290)
+- Enable e2e encryption for Pulsar Function [#8432](https://github.com/apache/pulsar/pull/8432)
+- Support key_based batch builder for functions and sources [#8523](https://github.com/apache/pulsar/pull/8523)
+- Refactor Context and State API to allow plugging different state store implementations [#8537](https://github.com/apache/pulsar/pull/8537)
+
+#### IO Connectors
```
Comment (Huanli-Meng) [inline site2/website/release-notes.md:116]:
```suggestion
#### IO connectors
```

File: site2/website/release-notes.md [hunk 3]
```diff
@@ -1,11 +1,180 @@
...
+- [Broker] Ensure that admin operations are gated by super user check [#7226](https://github.com/apache/pulsar/pull/7226)
+- [Broker] Fix race condition when delete topic forcelly [#7356](https://github.com/apache/pulsar/pull/7356)
+- [Tiered Storage] Fix NPE when offload data to GCS [#7400](https://github.com/apache/pulsar/pull/7400)
+- [Function]Fix race condition in which exitFuture in FunctionAssignmentTailer never gets completed even though the tailer thread has exited [#7351](https://github.com/apache/pulsar/pull/7351)
+- [Function] Various fixes and optimizations for processing assignments in function worker [#7338](https://github.com/apache/pulsar/pull/7338)
+- [Function] Fixed deadlock between create function and leader initialization [#7508](https://github.com/apache/pulsar/pull/7508)
+- [Pulsar Admin] Fix exceptions being ignored in PulsarAdmin [#7510](https://github.com/apache/pulsar/pull/7510)
+- [Broker] Fix the nondurable consumer can not specify the initial position [#7702](https://github.com/apache/pulsar/pull/7702)
+- [Broker] Fixed race condition on deleting topic with active readers [#7715](https://github.com/apache/pulsar/pull/7715)
```
Comment (Huanli-Meng) [inline site2/website/release-notes.md:165]:
```suggestion
- [Broker] Fix race condition on deleting topic with active readers [#7715](https://github.com/apache/pulsar/pull/7715)
```

File: site2/website/release-notes.md [hunk 4]
```diff
@@ -1,11 +1,180 @@
...
 ## Apache Pulsar Release Notes
 
-### 2.7.0 &mdash; Not Yet Released <a id=“2.7.0”></a>
+### 2.7.0 &mdash; 2020-11-25 <a id=“2.7.0”></a>
+
+The following lists fixes and enhancements in the 2.7.0 release.
+
+#### Transactions
+
+- Implementation of Transaction Buffer Client [#6544](https://github.com/apache/pulsar/pull/6544)
```
Comment (Huanli-Meng) [inline site2/website/release-notes.md:10]:
```suggestion
- Implement the transaction Buffer Client [#6544](https://github.com/apache/pulsar/pull/6544)
```

File: site2/website/release-notes.md [hunk 5]
```diff
@@ -1,11 +1,180 @@
...
+- Pending ack state implementation [#8426](https://github.com/apache/pulsar/pull/8426)
+- Support get reponse for message acknowledge [#8161](https://github.com/apache/pulsar/pull/8161)
+- Refactor the transaction buffer implementation [#8291](https://github.com/apache/pulsar/pull/8291) [#8347](https://github.com/apache/pulsar/pull/8347)
+- Transaction marker deletion [#8318](https://github.com/apache/pulsar/pull/8318)
+- Support produce messages with transaction in batch [#8415](https://github.com/apache/pulsar/pull/8415)
+- Register transaction metadata before send or ack messages [#8493](https://github.com/apache/pulsar/pull/8493)
+- Expose transaction interface [#8505](https://github.com/apache/pulsar/pull/8505)
+- Guarantee transaction metadata handlers connected [#8563](https://github.com/apache/pulsar/pull/8563)
+- Add the batch size in transaction ack command [#8659](https://github.com/apache/pulsar/pull/8659)
+- Transaction Log implementation [#8658](https://github.com/apache/pulsar/pull/8658)
```
Comment (Huanli-Meng) [inline site2/website/release-notes.md:26]:
```suggestion
- Implement the transaction log [#8658](https://github.com/apache/pulsar/pull/8658)
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. PTAL

**Review by eolivelli — APPROVED** [review]

**Pull request merged**